README.adoc

```diff
@@ -36,79 +35,94 @@ PowerPC::            NetBSD
 ARM::                NetBSD
 SPARC::              Solaris, Linux, NetBSD
 
-Other operating systems for the processors above have not been tested,
-but the compiler may work under other operating systems with little work.
+Other operating systems for the processors above have not been tested, but
+the compiler may work under other operating systems with little work.
 
-Before the introduction of objects, OCaml was known as Caml Special
-Light. OCaml is almost upwards compatible with Caml Special Light,
-except for a few additional reserved keywords that have forced some
-renaming of standard library functions.
+Before the introduction of objects, OCaml was known as Caml Special Light.
+OCaml is almost upwards compatible with Caml Special Light, except for a few
+additional reserved keywords that have forced some renaming of standard
+library functions.
 
-== CONTENTS
+== Contents
 
+  appveyor.yml::          configuration for appveyor CI
+  appveyor_build.sh::     appveyor CI script
   Changes::               what's new with each release
+  configure::             configure script
+  CONTRIBUTING.md::       how to contribute to OCaml
   INSTALL.adoc::          instructions for installation
   LICENSE::               license and copyright notice
   Makefile::              main Makefile
+  Makefile.nt::           MS Windows Makefile
+  Makefile.shared::       common Makefile
+  Makefile.tools::        used by manual/ and testsuite/ Makefiles
   README.adoc::           this file
-  README.win32.adoc::     infos on the MS Windows ports of OCaml
+  README.win32.adoc::     info on the MS Windows ports of OCaml
+  VERSION::               version string
   asmcomp/::              native-code compiler and linker
   asmrun/::               native-code runtime library
   boot/::                 bootstrap compiler
   bytecomp/::             bytecode compiler and linker
   byterun/::              bytecode interpreter and runtime system
+  compilerlibs/::         the OCaml compiler as a library
   config/::               autoconfiguration stuff
   debugger/::             source-level replay debugger
   driver/::               driver code for the compilers
   emacs/::                editing mode and debugger interface for GNU Emacs
+  experimental/::         experiments not built by default
```

Comment:
Well, I don't doubt that. :)
I agree they have historical values, but if everything is preserved in (stale) branches, it seems fine to me. And it would stop polluting `git grep` ...